src/lib/components/forms/form/form-components/terra-form-component-base.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface TerraFormComponentBaseInterface {
+    name: string;
+    isRequired: boolean;
+    tooltipText: string;
+    tooltipPlacement: string;
+    isDisabled: boolean;
```

Comment:
Was ist hiermit?
<img width="513" alt="Bildschirmfoto 2021-04-09 um 15 50 59" src="https://user-images.githubusercontent.com/26869118/114190312-75083080-994b-11eb-8ddc-3b1b56cd73be.png">

Reply:
wurde in der "alten" text-area komponente nicht genutzt

Reply:
bis auf `maxLength` stimmt das.
D.h. alle Components hatten zwar diese Inputs. Bei manchen hatten sie aber keine Funktion :D 🙈

Reply:
geklärt